From 7879277c0f7e163f3c6171b7aa809158853cd9f5 Mon Sep 17 00:00:00 2001
From: Jeremy Newton <Jeremy.Newton@amd.com>
Date: Mon, 10 Apr 2023 15:08:25 -0400
Subject: [PATCH 1/4] Test reporting error

The mangled_names_test reports the wrong value, which I found when
debugging test issues using upstream LLVM.
---
 lib/comgr/test/mangled_names_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/comgr/test/mangled_names_test.c b/lib/comgr/test/mangled_names_test.c
index 946a13d..352ad31 100644
--- a/lib/comgr/test/mangled_names_test.c
+++ b/lib/comgr/test/mangled_names_test.c
@@ -252,7 +252,7 @@ int main(int argc, char *argv[]) {
   if (numNames != 4) {
     printf("amd_populate_mangled_names Failed: "
            "produced %zu executable names (expected 4)\n",
-           Count);
+           numNames);
     exit(1);
   }
 
-- 
2.39.2

